browser-test/src/support/index.ts

```diff
@@ -539,6 +539,33 @@ export const validateScreenshot = async (
   }
 
   expect(screenshotFileName).toMatch(/^[a-z0-9-]+$/)
+
+  await takeScreenshot(element, `${screenshotFileName}`, fullPage)
+
+  const existingWidth = page.viewportSize()?.width || 1280
+
+  if (mobileScreenshot) {
```

Comment:
i thought about that, but I kind of like still using mobileScreenshot since I think it makes it more clear to callers that it's something to include on views that need to be mobile friendly. The medium width is still kinda mobile it's just more like a tablet or something.